packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -619,12 +704,29 @@ class _TableRowState<T> extends State<TableRow<T>>
       Widget content;
       final node = widget.node;
       if (node == null) {
-        content = Text(
-          column.title,
-          overflow: TextOverflow.ellipsis,
+        final isSortColumn = column.title == widget.sortColumn.title;
+        content = InkWell(
+          onTap: () => _handleSortChange(column),
+          child: Row(
+            children: [
+              if (isSortColumn)
+                Icon(
+                  widget.sortDirection == SortDirection.ascending
+                      ? Icons.expand_less
+                      : Icons.expand_more,
+                  size: 16.0,
```

Comment:
we shouldn't be leaking our default icon size all over the place. add this as a constant in the theme and cleanup other cases where we are hard coding size 16 assuming we are.